client/components/banner/style.scss

```diff
@@ -217,6 +217,11 @@
 		}
 	}
 
+	a,
+	button {
+		white-space: nowrap;
+	}
```

Comment:
I am really worried that this may introduce unexpected behaviour elsewhere in Calypso, so I don't think we should add these styles to the banner component in this PR.

If this style is really needed, then I think we should add a `className` to the banner and add the style specifically for our context.

Reply:
yes, good point! I think we need it to avoid weird text wrapping on the button, but I will make it specific to this ConciergeBanner.